hikari/impl/entity_factory.py

```diff
@@ -704,6 +705,8 @@ def deserialize_guild_voice_channel(
         guild_id: undefined.UndefinedOr[snowflakes.Snowflake] = undefined.UNDEFINED,
     ) -> channel_models.GuildVoiceChannel:
         channel_fields = self._set_guild_channel_attributes(payload, guild_id=guild_id)
+        # Discord seems to be only returning this after it's been initially PATCHed in for older channels.
+        video_quality_mode = payload.get("video_quality_mode", channel_models.VideoQualityMode.AUTO)
```

Comment:
tbh kind of surprised that i missed this behaviour while implementing it, guess i'll have to expect this stuff from now on since we already got similar behaviour with guild member screening and welcome screens